### ISSUE TITLE
Error: Cannot find module 'babel-preset-env' from '/app'

### ISSUE THREAD

**Issue by foxsys-admin** [cookiecutter-flask/cookiecutter-flask#666]
Hi, first of all, congratulations for this great project! 

I have cloned this repo from master branch and I get an error when I execute `docker-compose up flask-prod`. By the way, I have cloned the master branch, maybe it's the development branch and it's not intended for production use. Maybe I should use [one of the releases instead](https://github.com/cookiecutter-flask/cookiecutter-flask/releases). Maybe this error is related to [this recent commit](https://github.com/cookiecutter-flask/cookiecutter-flask/commit/da24a03f50d0dc016a97cca1159b8576ff74862a). Ok, hope it helps :) 

This is the output, thanks in advance!

```
flask-prod_1  | Entrypoint main_css = main_css.bundle.css main_css.bundle.js
flask-prod_1  | [0] ./assets/js/main.js 2.24 KiB {1} [built] [failed] [1 error]
flask-prod_1  | [1] multi ./node_modules/font-awesome/css/font-awesome.css ./node_modules/bootstrap/dist/css/bootstrap.css ./assets/css/style.css 52 bytes {0} [built]
flask-prod_1  | [4] ./assets/css/style.css 39 bytes {0} [built]
flask-prod_1  |     + 5 hidden modules
flask-prod_1  | 
flask-prod_1  | ERROR in ./assets/js/main.js
flask-prod_1  | Module build failed (from ./node_modules/babel-loader/lib/index.js):
flask-prod_1  | Error: Cannot find module 'babel-preset-env' from '/app'
flask-prod_1  | - Did you mean "@babel/env"?
flask-prod_1  |     at Function.module.exports [as sync] (/app/node_modules/resolve/lib/sync.js:74:15)
flask-prod_1  |     at resolveStandardizedName (/app/node_modules/@babel/core/lib/config/files/plugins.js:101:31)
flask-prod_1  |     at resolvePreset (/app/node_modules/@babel/core/lib/config/files/plugins.js:58:10)
flask-prod_1  |     at loadPreset (/app/node_modules/@babel/core/lib/config/files/plugins.js:77:20)
flask-prod_1  |     at createDescriptor (/app/node_modules/@babel/core/lib/config/config-descriptors.js:154:9)
flask-prod_1  |     at /app/node_modules/@babel/core/lib/config/config-descriptors.js:109:50
flask-prod_1  |     at Array.map (<anonymous>)
flask-prod_1  |     at createDescriptors (/app/node_modules/@babel/core/lib/config/config-descriptors.js:109:29)
flask-prod_1  |     at createPresetDescriptors (/app/node_modules/@babel/core/lib/config/config-descriptors.js:101:10)
flask-prod_1  |     at /app/node_modules/@babel/core/lib/config/config-descriptors.js:58:96
flask-prod_1  |     at cachedFunction (/app/node_modules/@babel/core/lib/config/caching.js:32:19)
flask-prod_1  |     at presets (/app/node_modules/@babel/core/lib/config/config-descriptors.js:29:84)
flask-prod_1  |     at mergeChainOpts (/app/node_modules/@babel/core/lib/config/config-chain.js:320:26)
flask-prod_1  |     at /app/node_modules/@babel/core/lib/config/config-chain.js:283:7
flask-prod_1  |     at buildRootChain (/app/node_modules/@babel/core/lib/config/config-chain.js:68:29)
flask-prod_1  |     at loadPrivatePartialConfig (/app/node_modules/@babel/core/lib/config/partial.js:85:55)
flask-prod_1  |     at Object.loadPartialConfig (/app/node_modules/@babel/core/lib/config/partial.js:110:18)
flask-prod_1  |     at Object.<anonymous> (/app/node_modules/babel-loader/lib/index.js:144:26)
flask-prod_1  |     at Generator.next (<anonymous>)
flask-prod_1  |     at asyncGeneratorStep (/app/node_modules/babel-loader/lib/index.js:3:103)
flask-prod_1  |     at _next (/app/node_modules/babel-loader/lib/index.js:5:194)
flask-prod_1  |     at /app/node_modules/babel-loader/lib/index.js:5:364
flask-prod_1  |     at new Promise (<anonymous>)
flask-prod_1  |     at Object.<anonymous> (/app/node_modules/babel-loader/lib/index.js:5:97)
flask-prod_1  |     at Object._loader (/app/node_modules/babel-loader/lib/index.js:224:18)
flask-prod_1  |     at Object.loader (/app/node_modules/babel-loader/lib/index.js:60:18)
flask-prod_1  |     at Object.<anonymous> (/app/node_modules/babel-loader/lib/index.js:55:12)
flask-prod_1  | Child mini-css-extract-plugin node_modules/css-loader/dist/cjs.js!assets/css/style.css:
flask-prod_1  |     Entrypoint mini-css-extract-plugin = *
flask-prod_1  |     [0] ./node_modules/css-loader/dist/cjs.js!./assets/css/style.css 1.46 KiB {0} [built]
flask-prod_1  |         + 1 hidden module
flask-prod_1  | Child mini-css-extract-plugin node_modules/css-loader/dist/cjs.js!node_modules/bootstrap/dist/css/bootstrap.css:
flask-prod_1  |     Entrypoint mini-css-extract-plugin = *
flask-prod_1  |        2 modules
flask-prod_1  | Child mini-css-extract-plugin node_modules/css-loader/dist/cjs.js!node_modules/font-awesome/css/font-awesome.css:
flask-prod_1  |     Entrypoint mini-css-extract-plugin = *
flask-prod_1  |        9 modules
flask-prod_1  | npm ERR! code ELIFECYCLE
flask-prod_1  | npm ERR! errno 2
flask-prod_1  | npm ERR! web@1.0.0 build: `NODE_ENV=production webpack --progress --colors -p && npm run flask-static-digest`
flask-prod_1  | npm ERR! Exit status 2
flask-prod_1  | npm ERR! 
flask-prod_1  | npm ERR! Failed at the web@1.0.0 build script.
flask-prod_1  | npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
flask-prod_1  | 
flask-prod_1  | npm ERR! A complete log of this run can be found in:
flask-prod_1  | npm ERR!     /home/sid/.npm/_logs/2019-12-25T22_57_57_714Z-debug.log
web_flask-prod_1 exited with code 2
```

**Comment by jamescurtin** [comment]
Hi--thanks for the report! Would you be able to let me know which of the options you selected when generating the cookiecutter? 


|      Option      |                                Values                                 |
|------------------|-----------------------------------------------------------------------|
| `use_pipenv`     | <ul><li>- [ ] `yes`</li><li>- [ ] `no`</li></ul>                      |
| `python_version` | <ul><li>- [ ] `3.8`</li><li>- [ ] `3.7`</li><li>- [ ] `3.6`</li></ul> |
| `node_version`   | <ul><li>- [ ] `12`</li><li>- [ ] `10`</li></ul>                       |
| `use_heroku`     | <ul><li>- [ ] `yes`</li><li>- [ ] `no`</li></ul>                      |

---

When I create a fresh version of the project (`use_pipenv`: `no`, `python_version`: `3.7`, `node_version`: `12`, `use_heroku`: `no`), I am not able to reproduce this error. Based on the error message, it looks like your project may have an older version of a dependency pinned; have you generated an app using an older version of this project in the past? I am wondering if the container `flask-prod_1` is mounting the `node_modules` volume that was created using a previous run, and it hasn't been updated.

To test this hypothesis, would you be able to run the following (assuming the name of your project is `my_flask_app`):

```bash
$ docker volume rm my_flask_app_node-modules
$ docker-compose down
$ docker-compose build flask-prod
$ docker-compose up flask-prod
```

**Comment by foxsys-admin** [comment]
@jamescurtin Hi James, thank you very much for your promptly response! I have tried that approach (starting from scratch) by deleting all docker containers and docker volumes and then executing "docker-compose build flask-prod" and "docker-compoes up flask-prod" using the same settings you have mentioned above, but I'm still facing the same error. I'm executing these steps on Linux Mint 19.3. Let me explain what I did today, step by step:

```
$ cd /tmp

$ git clone https://github.com/cookiecutter-flask/cookiecutter-flask

$ cd cookiecutter-flask/

$ ./cookiecutter-docker.sh

full_name [Steven Loria]: FoxSys

email [sloria1@gmail.com]: <obfuscated>

github_username [sloria]: foxsys-admin

project_name [My Flask App]: Projectix

app_name [projectix]: app

project_short_description [A flasky app.]: Projects Management

Select use_pipenv:
1 - no
2 - yes
Choose from 1, 2 (1, 2) [1]: 

Select python_version:
1 - 3.8
2 - 3.7
3 - 3.6
Choose from 1, 2, 3 (1, 2, 3) [1]: 2

Select node_version:
1 - 12
2 - 10
Choose from 1, 2 (1, 2) [1]: 

Select use_heroku:
1 - no
2 - yes
Choose from 1, 2 (1, 2) [1]: 
```

Ok, after executing the cookiecutter-docker.sh command, I have performed the docker compose build:

```
$ cd app

$ docker-compose build flask-prod

The build process is smooth, I just got a warning, sharing here just in case it's important:

[...]

Step 15/27 : RUN npm install
 ---> Running in 478e9ff3fbe5
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.11 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.11: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

[...]
```

And finally I start the docker compose stack, where I get the same error:

```
$ docker-compose up flask-prod

Creating network "app_default" with the default driver
Creating volume "app_node-modules" with default driver
Creating volume "app_static-build" with default driver
Creating volume "app_dev-db" with default driver
Creating app_flask-prod_1 ... done
Attaching to app_flask-prod_1
flask-prod_1  | 
flask-prod_1  | > app@1.0.0 build /app
flask-prod_1  | > NODE_ENV=production webpack --progress --colors -p && npm run flask-static-digest
flask-prod_1  | 
flask-prod_1  | <s> [webpack.Progress] 0% compiling
flask-prod_1  | <s> [webpack.Progress] 10% building 0/0 modules 0 active 
flask-prod_1  | <s> [webpack.Progress] 10% building 0/0 modules 0 active 
flask-prod_1  | <s> [webpack.Progress] 10% building 0/1 modules 1 active multi /app/node_modules/font-awesome/css/font-awesome.css /app/node_modules/bootstrap/dist/css/bootstrap.css /app/assets/css/style.css
flask-prod_1  | <s> [webpack.Progress] 10% building 1/1 modules 0 active 
flask-prod_1  | <s> [webpack.Progress] 10% building 1/2 modules 1 active /app/node_modules/mini-css-extract-plugin/dist/loader.js??ref--5-0!/app/node_modules/css-loader/dist/cjs.js!/app/assets/css/style.css
flask-prod_1  | <s> [webpack.Progress] 10% building 1/3 modules 2 active /app/node_modules/mini-css-extract-plugin/dist/loader.js??ref--5-0!/app/node_modules/css-loader/dist/cjs.js!/app/node_modules/font-awesome/css/font-awesome.css
flask-prod_1  | <s> [webpack.Progress] 10% building 1/4 modules 3 active /app/node_modules/mini-css-extract-plugin/dist/loader.js??ref--5-0!/app/node_modules/css-loader/dist/cjs.js!/app/node_modules/bootstrap/dist/css/bootstrap.css
flask-prod_1  | <s> [webpack.Progress] 10% building 1/5 modules 4 active /app/node_modules/babel-loader/lib/index.js??ref--9!/app/assets/js/main.js
flask-prod_1  | <s> [webpack.Progress] 10% building 2/5 modules 3 active /app/node_modules/babel-loader/lib/index.js??ref--9!/app/assets/js/main.js
flask-prod_1  | <s> [webpack.Progress] 10% building 2/6 modules 4 active css /app/node_modules/css-loader/dist/cjs.js!/app/assets/css/style.css 0
flask-prod_1  | <s> [webpack.Progress] 10% building 3/6 modules 3 active /app/node_modules/babel-loader/lib/index.js??ref--9!/app/assets/js/main.js
flask-prod_1  | <s> [webpack.Progress] 10% building 4/6 modules 2 active /app/node_modules/babel-loader/lib/index.js??ref--9!/app/assets/js/main.js
flask-prod_1  | <s> [webpack.Progress] 10% building 4/7 modules 3 active css /app/node_modules/css-loader/dist/cjs.js!/app/node_modules/bootstrap/dist/css/bootstrap.css 0
flask-prod_1  | <s> [webpack.Progress] 10% building 5/7 modules 2 active /app/node_modules/babel-loader/lib/index.js??ref--9!/app/assets/js/main.js
flask-prod_1  | <s> [webpack.Progress] 10% building 6/7 modules 1 active /app/node_modules/mini-css-extract-plugin/dist/loader.js??ref--5-0!/app/node_modules/css-loader/dist/cjs.js!/app/node_modules/font-awesome/css/font-awesome.css
flask-prod_1  | <s> [webpack.Progress] 40% building 6/7 modules 1 active /app/node_modules/mini-css-extract-plugin/dist/loader.js??ref--5-0!/app/node_modules/css-loader/dist/cjs.js!/app/node_modules/font-awesome/css/font-awesome.css
flask-prod_1  | <s> [webpack.Progress] 40% building 7/7 modules 0 active 
flask-prod_1  | <s> [webpack.Progress] 40% building 7/8 modules 1 active css /app/node_modules/css-loader/dist/cjs.js!/app/node_modules/font-awesome/css/font-awesome.css 0
flask-prod_1  | <s> [webpack.Progress] 40% building 8/8 modules 0 active 
flask-prod_1  | <s> [webpack.Progress] 70% building 8/8 modules 0 active 
flask-prod_1  | <s> [webpack.Progress] 70% finish module graph
flask-prod_1  | <s> [webpack.Progress] 70% finish module graph FlagDependencyExportsPlugin
flask-prod_1  | <s> [webpack.Progress] 70% finish module graph WasmFinalizeExportsPlugin
flask-prod_1  | <s> [webpack.Progress] 70% sealing
flask-prod_1  | <s> [webpack.Progress] 70% sealing WarnCaseSensitiveModulesPlugin
flask-prod_1  | <s> [webpack.Progress] 72% basic dependencies optimization
flask-prod_1  | <s> [webpack.Progress] 72% dependencies optimization
flask-prod_1  | <s> [webpack.Progress] 72% dependencies optimization SideEffectsFlagPlugin
flask-prod_1  | <s> [webpack.Progress] 72% dependencies optimization FlagDependencyUsagePlugin
flask-prod_1  | <s> [webpack.Progress] 73% advanced dependencies optimization
flask-prod_1  | <s> [webpack.Progress] 73% after dependencies optimization
flask-prod_1  | <s> [webpack.Progress] 71% chunk graph
flask-prod_1  | <s> [webpack.Progress] 71% after chunk graph
flask-prod_1  | <s> [webpack.Progress] 71% after chunk graph WebAssemblyModulesPlugin
flask-prod_1  | <s> [webpack.Progress] 74% optimizing
flask-prod_1  | <s> [webpack.Progress] 74% basic module optimization
flask-prod_1  | <s> [webpack.Progress] 75% module optimization
flask-prod_1  | <s> [webpack.Progress] 75% advanced module optimization
flask-prod_1  | <s> [webpack.Progress] 76% after module optimization
flask-prod_1  | <s> [webpack.Progress] 76% basic chunk optimization
flask-prod_1  | <s> [webpack.Progress] 76% basic chunk optimization EnsureChunkConditionsPlugin
flask-prod_1  | <s> [webpack.Progress] 76% basic chunk optimization RemoveParentModulesPlugin
flask-prod_1  | <s> [webpack.Progress] 76% basic chunk optimization RemoveEmptyChunksPlugin
flask-prod_1  | <s> [webpack.Progress] 76% basic chunk optimization MergeDuplicateChunksPlugin
flask-prod_1  | <s> [webpack.Progress] 77% chunk optimization
flask-prod_1  | <s> [webpack.Progress] 77% advanced chunk optimization
flask-prod_1  | <s> [webpack.Progress] 77% advanced chunk optimization SplitChunksPlugin
flask-prod_1  | <s> [webpack.Progress] 77% advanced chunk optimization RemoveEmptyChunksPlugin
flask-prod_1  | <s> [webpack.Progress] 77% after chunk optimization
flask-prod_1  | <s> [webpack.Progress] 78% module and chunk tree optimization
flask-prod_1  | <s> [webpack.Progress] 78% after module and chunk tree optimization
flask-prod_1  | <s> [webpack.Progress] 79% basic chunk modules optimization
flask-prod_1  | <s> [webpack.Progress] 80% chunk modules optimization
flask-prod_1  | <s> [webpack.Progress] 80% chunk modules optimization ModuleConcatenationPlugin
flask-prod_1  | <s> [webpack.Progress] 80% advanced chunk modules optimization
flask-prod_1  | <s> [webpack.Progress] 81% after chunk modules optimization
flask-prod_1  | <s> [webpack.Progress] 81% module reviving
flask-prod_1  | <s> [webpack.Progress] 81% module reviving RecordIdsPlugin
flask-prod_1  | <s> [webpack.Progress] 82% module order optimization
flask-prod_1  | <s> [webpack.Progress] 82% module order optimization OccurrenceOrderModuleIdsPlugin
flask-prod_1  | <s> [webpack.Progress] 82% advanced module order optimization
flask-prod_1  | <s> [webpack.Progress] 83% before module ids
flask-prod_1  | <s> [webpack.Progress] 83% module ids
flask-prod_1  | <s> [webpack.Progress] 84% module id optimization
flask-prod_1  | <s> [webpack.Progress] 84% module id optimization
flask-prod_1  | <s> [webpack.Progress] 85% chunk reviving
flask-prod_1  | <s> [webpack.Progress] 85% chunk reviving RecordIdsPlugin
flask-prod_1  | <s> [webpack.Progress] 85% chunk order optimization
flask-prod_1  | <s> [webpack.Progress] 85% chunk order optimization OccurrenceOrderChunkIdsPlugin
flask-prod_1  | <s> [webpack.Progress] 86% before chunk ids
flask-prod_1  | <s> [webpack.Progress] 86% chunk id optimization
flask-prod_1  | <s> [webpack.Progress] 86% chunk id optimization FlagIncludedChunksPlugin
flask-prod_1  | <s> [webpack.Progress] 87% after chunk id optimization
flask-prod_1  | <s> [webpack.Progress] 88% hashing
flask-prod_1  | <s> [webpack.Progress] 88% after hashing
flask-prod_1  | <s> [webpack.Progress] 89% module assets processing
flask-prod_1  | <s> [webpack.Progress] 90% chunk assets processing
flask-prod_1  | <s> [webpack.Progress] 90% additional chunk assets processing
flask-prod_1  | <s> [webpack.Progress] 92% additional asset processing
flask-prod_1  | <s> [webpack.Progress] 92% chunk asset optimization
flask-prod_1  | <s> [webpack.Progress] 92% chunk asset optimization TerserPlugin
flask-prod_1  | <s> [webpack.Progress] 93% after chunk asset optimization
flask-prod_1  | <s> [webpack.Progress] 93% asset optimization
flask-prod_1  | <s> [webpack.Progress] 94% after asset optimization
flask-prod_1  | <s> [webpack.Progress] 94% after seal
flask-prod_1  | <s> [webpack.Progress] 100% 
flask-prod_1  | 
flask-prod_1  | Hash: fbdca7e9e7eb0c87a25b
flask-prod_1  | Version: webpack 4.41.4
flask-prod_1  | Time: 904ms
flask-prod_1  | Built at: 12/26/2019 8:16:05 PM
flask-prod_1  |  8 assets
flask-prod_1  | Entrypoint main_js = main_js.bundle.js
flask-prod_1  | Entrypoint main_css = main_css.bundle.css main_css.bundle.js
flask-prod_1  | [0] ./assets/js/main.js 2.24 KiB {1} [built] [failed] [1 error]
flask-prod_1  | [1] multi ./node_modules/font-awesome/css/font-awesome.css ./node_modules/bootstrap/dist/css/bootstrap.css ./assets/css/style.css 52 bytes {0} [built]
flask-prod_1  | [4] ./assets/css/style.css 39 bytes {0} [built]
flask-prod_1  |     + 5 hidden modules
flask-prod_1  | 
flask-prod_1  | ERROR in ./assets/js/main.js
flask-prod_1  | Module build failed (from ./node_modules/babel-loader/lib/index.js):
flask-prod_1  | Error: Cannot find module 'babel-preset-env' from '/app'
flask-prod_1  | - Did you mean "@babel/env"?
flask-prod_1  |     at Function.module.exports [as sync] (/app/node_modules/resolve/lib/sync.js:74:15)
flask-prod_1  |     at resolveStandardizedName (/app/node_modules/@babel/core/lib/config/files/plugins.js:101:31)
flask-prod_1  |     at resolvePreset (/app/node_modules/@babel/core/lib/config/files/plugins.js:58:10)
flask-prod_1  |     at loadPreset (/app/node_modules/@babel/core/lib/config/files/plugins.js:77:20)
flask-prod_1  |     at createDescriptor (/app/node_modules/@babel/core/lib/config/config-descriptors.js:154:9)
flask-prod_1  |     at /app/node_modules/@babel/core/lib/config/config-descriptors.js:109:50
flask-prod_1  |     at Array.map (<anonymous>)
flask-prod_1  |     at createDescriptors (/app/node_modules/@babel/core/lib/config/config-descriptors.js:109:29)
flask-prod_1  |     at createPresetDescriptors (/app/node_modules/@babel/core/lib/config/config-descriptors.js:101:10)
flask-prod_1  |     at /app/node_modules/@babel/core/lib/config/config-descriptors.js:58:96
flask-prod_1  |     at cachedFunction (/app/node_modules/@babel/core/lib/config/caching.js:32:19)
flask-prod_1  |     at presets (/app/node_modules/@babel/core/lib/config/config-descriptors.js:29:84)
flask-prod_1  |     at mergeChainOpts (/app/node_modules/@babel/core/lib/config/config-chain.js:320:26)
flask-prod_1  |     at /app/node_modules/@babel/core/lib/config/config-chain.js:283:7
flask-prod_1  |     at buildRootChain (/app/node_modules/@babel/core/lib/config/config-chain.js:68:29)
flask-prod_1  |     at loadPrivatePartialConfig (/app/node_modules/@babel/core/lib/config/partial.js:85:55)
flask-prod_1  |     at Object.loadPartialConfig (/app/node_modules/@babel/core/lib/config/partial.js:110:18)
flask-prod_1  |     at Object.<anonymous> (/app/node_modules/babel-loader/lib/index.js:144:26)
flask-prod_1  |     at Generator.next (<anonymous>)
flask-prod_1  |     at asyncGeneratorStep (/app/node_modules/babel-loader/lib/index.js:3:103)
flask-prod_1  |     at _next (/app/node_modules/babel-loader/lib/index.js:5:194)
flask-prod_1  |     at /app/node_modules/babel-loader/lib/index.js:5:364
flask-prod_1  |     at new Promise (<anonymous>)
flask-prod_1  |     at Object.<anonymous> (/app/node_modules/babel-loader/lib/index.js:5:97)
flask-prod_1  |     at Object._loader (/app/node_modules/babel-loader/lib/index.js:224:18)
flask-prod_1  |     at Object.loader (/app/node_modules/babel-loader/lib/index.js:60:18)
flask-prod_1  |     at Object.<anonymous> (/app/node_modules/babel-loader/lib/index.js:55:12)
flask-prod_1  | Child mini-css-extract-plugin node_modules/css-loader/dist/cjs.js!assets/css/style.css:
flask-prod_1  |     Entrypoint mini-css-extract-plugin = *
flask-prod_1  |     [0] ./node_modules/css-loader/dist/cjs.js!./assets/css/style.css 1.46 KiB {0} [built]
flask-prod_1  |         + 1 hidden module
flask-prod_1  | Child mini-css-extract-plugin node_modules/css-loader/dist/cjs.js!node_modules/bootstrap/dist/css/bootstrap.css:
flask-prod_1  |     Entrypoint mini-css-extract-plugin = *
flask-prod_1  |        2 modules
flask-prod_1  | Child mini-css-extract-plugin node_modules/css-loader/dist/cjs.js!node_modules/font-awesome/css/font-awesome.css:
flask-prod_1  |     Entrypoint mini-css-extract-plugin = *
flask-prod_1  |        9 modules
flask-prod_1  | npm ERR! code ELIFECYCLE
flask-prod_1  | npm ERR! errno 2
flask-prod_1  | npm ERR! app@1.0.0 build: `NODE_ENV=production webpack --progress --colors -p && npm run flask-static-digest`
flask-prod_1  | npm ERR! Exit status 2
flask-prod_1  | npm ERR! 
flask-prod_1  | npm ERR! Failed at the app@1.0.0 build script.
flask-prod_1  | npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
flask-prod_1  | 
flask-prod_1  | npm ERR! A complete log of this run can be found in:
flask-prod_1  | npm ERR!     /home/sid/.npm/_logs/2019-12-26T20_16_05_829Z-debug.log
app_flask-prod_1 exited with code 2
```

What do you think? Thanks again for your help!

**Comment by foxsys-admin** [comment]
@jamescurtin I did some research using Google and I found this issue. I have applied the same solution on webpack.config.js, replacing `{ "presets": ["env"],` with `{ "presets": ["@babel/preset-env"],` and it worked. Now I can execute `docker-compose up flask-prod` without errors. The frontpage can be viewed correctly. Just in case it helps to others, I have opened a [pull request](https://github.com/cookiecutter-flask/cookiecutter-flask/pull/668). Thanks!
